frontend/android/src/main/java/com/jakewharton/sdksearch/ui/ItemViewHolder.kt

```diff
@@ -93,12 +95,33 @@ internal class ItemViewHolder(
     val query = this.query!!
     val item = this.item!!
 
-    packageNameText.text = item.package_()
+    packageNameText.text = buildSpannedString {
+      val splitIt = item.package_().split('.').iterator()
```

Comment:
This can be done with `forEachIndexed` to avoid having to interact with the iterator directly:
```kotlin
item.package_().split('.').forEachIndexed { index, part ->
  if (index > 0) {
    // span + period
  }
  append(part)
}
```

Reply:
Nice. I was trying to think of a better way (like a `indicesOf()` which returns all indices)